build.cmd

```diff
@@ -72,7 +72,7 @@ setlocal
   cd /d %~dp0tests\simpleharness
   call :dotnet_build || exit /b 1
 
-  for %%v in (netcoreapp1.0 netcoreapp1.1 netcoreapp2.0 net461) do (
+  for %%v in (netcoreapp2.0 net461) do (
```

Comment:
Should we add netcoreapp2.1 testing too, and download latest tooling too: 2.1.300?

Reply:
I don't have a strong opinion here but I think netcoreapp2.0 is enough for the test matrix.